src/test/java/org/fcrepo/importexport/integration/ExporterIT.java

```diff
@@ -317,6 +321,44 @@ public void testExportCustomPredicatesInbound() throws Exception {
                 createResource(obj1.toString()),
                 createProperty("http://example.org/custom"),
                 createResource(col1.toString())));
+        assertFalse(collectionModel.contains(
+                createResource(other.toString()),
+                createProperty("http://example.org/other"),
+                createResource(col1.toString())));
+    }
+
+    @Test
+    public void testExportInboundFiles() throws Exception {
```

Comment:
Since the test is less related to the rest of the PR, pulling into a separate branch/PR targeting [fcrepo-2500](https://jira.duraspace.org/browse/FCREPO-2500) makes sense.

What are your thoughts on an integration test (potentially as described above) exercising the fix in this current PR?